docs/verification.md

```diff
@@ -51,6 +51,8 @@ You can bypass this check only if both of the following conditions meet.
 - `allow_no_verification = true` is specified in `config.toml` of stargz snapshotter, and
 - the content descriptor of this layer has an annotation `containerd.io/snapshot/remote/stargz.skipverify` (the value will be ignored).
 
+The other way is to disable verification completely by setting `disable_verification = true` in `config.toml` of stargz snapshotter. This option is necessary when working with stargz (non-eStargz) format.
```

Comment:
> The other way is to disable verification completely by setting `disable_verification = true` in `config.toml` of stargz snapshotter.

sounds good to me.

> This option is necessary when working with stargz (non-eStargz) format.

This isn't accurate because we can currently work with stargz (non-eStargz) format when we use `ctr-remote` command, even without `disable_verification`. Maybe we don't need this sentence here.

Reply:
I removed the sentence.